src/node_task_runner.cc

```diff
@@ -0,0 +1,219 @@
+#include "node_task_runner.h"
+#include "util.h"
+
+#include <regex>  // NOLINT(build/c++11)
+
+namespace node::task_runner {
+
+#if defined(_WIN32)
```

Comment:
```suggestion
#ifdef _WIN32
```
Can we do this or is the proposed one used for consistency reasons?

Reply:
Yeah, this is better. I'll apply. Thanks!